lib/orderbook/MatchingEngine.ts

```diff
@@ -37,7 +38,7 @@ class MatchingEngine {
   /** a pair of maps between active own orders ids and orders for the buy and sell sides of this trading pair. */
   public ownOrders: OrderSidesLists<orders.StampedOwnOrder>;
   /** a pair of maps between active peer orders ids and orders for the buy and sell sides of this trading pair. */
```

Comment:
You can use the updated comment: `  /** a map between peerPubKey and a pair of maps between active peer orders ids and orders for the buy and sell sides of this trading pair. */`

Reply:
Minor nit if we're changing this anyway, but these comments should start with capital letters.